src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/item/MagitechGogglesItem.java

```diff
@@ -78,7 +78,6 @@ public MagitechGogglesItem() {
         return ImmutableMultimap.of(AMAttributes.MAGIC_VISION.get(), new AttributeModifier(UUID, "magic_vision", 1, AttributeModifier.Operation.ADDITION));
     }
 
-    @Nullable
```

Comment:
isn't the super method also nullable?

Reply:
Exactly. IntelliJ suggested removing it as the nullability is already defined by the superclass.